File: rust/src/models/number_config_value.rs
// this file is @generated
use serde::{Deserialize, Serialize};

/// A number config value (decimal, encoded as a string).
#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct NumberConfigValue {
    pub value: rust_decimal::Decimal,
}

impl NumberConfigValue {
    pub fn new(value: rust_decimal::Decimal) -> Self {
        Self { value }
    }
}

File: browser/src/models/numberConfigValue.ts
// this file is @generated
/** A number config value (decimal, encoded as a string). */
export interface NumberConfigValue {
  value: string;
}

export const NumberConfigValueSerializer = {
  _fromJsonObject(object: any): NumberConfigValue {
    return {
      value: object["value"],
    };
  },

  _toJsonObject(self: NumberConfigValue): any {
    return {
      value: self.value,
    };
  },
};
